pkg/workloads/cortex/lib/log.py

```diff
@@ -28,23 +29,39 @@ def formatTime(self, record, datefmt):
         return s
 
 
-logger = logging.getLogger("cortex")
-handler = logging.StreamHandler(stream=sys.stdout)
-formatter = MyFormatter(
-    fmt="%(asctime)s:%(name)s:%(levelname)s:%(message)s", datefmt="%Y-%m-%d %H:%M:%S.%f"
-)
-handler.setFormatter(formatter)
+current_logger = None
 
-logger.addHandler(handler)
-logger.setLevel(logging.DEBUG)
+
+def register_logger(name):
+    logger = logging.getLogger(name)
+    handler = logging.StreamHandler(stream=sys.stdout)
+    formatter = MyFormatter(
+        fmt="%(asctime)s:cortex:%(levelname)s:%(message)s", datefmt="%Y-%m-%d %H:%M:%S.%f"
+    )
+    handler.setFormatter(formatter)
+
+    logger.propagate = False
+    logger.addHandler(handler)
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+def refresh_logger():
+    global current_logger
+    if current_logger is not None:
+        current_logger.disabled = True
+    current_logger = register_logger("{}-cortex".format(int(time.time() * 1000000)))
+
+
+def cx_logger():
```

Comment:
Is it possible / is it a good idea / is it worth it to implement each of the logger methods to wrap `current_logger`? E.g. if we had something like

```python
def info(str):
    return current_logger.info(str)
```

then could we do `import cx_logger` and use `cx_logger.info()`? We can discuss